sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb

```diff
@@ -966,7 +966,7 @@
     "scores = pd.read_csv(\"stock_sec_scores.csv\")\n",
```

Comment:
<p>Why we are decreasing it? Is there any benefit of reducing it to 5?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
5 is an arbitrary number. The main point is the number has to be less than 10 because the dataframe only has 10 rows. Th original code picked 11, which resulted in out of index error